packages/app/src/components/CustomizePracticeCard.vue

```diff
@@ -186,7 +190,7 @@ export default {
 }
 #btn-customize-practice {
   z-index: 4;
-  color: gray !important;
+  color: white !important;
```

Comment:
What I meant was the color of the icon should be #114489, and the background should remain white :)